src/main/resources/templates/application/summary/proceedings-matter-type.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en"
+      th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}"
+      xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+  <title>LAA CCMS - Create application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a href="/application/summary/proceedings-and-costs" class="govuk-back-link">Cancel and return to Proceedings and costs</a>
```

Comment:
Hmmmm, maybe the lower case ones were the odd one out :-)

Reply:
amended